initrd/bin/kexec-seal-key

```diff
@@ -26,87 +26,111 @@ KEY_LVM="$paramsdir/kexec_key_lvm.txt"
 
 if [ ! -r "$KEY_DEVICES" ]; then
 	die "No devices defined for disk encryption"
+else
+	DEBUG "Devices defined for disk encryption: $(cat "$KEY_DEVICES" | cut -d\  -f1)"
 fi
 
 if [ -r "$KEY_LVM" ]; then
 	# Activate the LVM volume group
-	VOLUME_GROUP=`cat $KEY_LVM`
+	VOLUME_GROUP=$(cat $KEY_LVM)
 	if [ -z "$VOLUME_GROUP" ]; then
 		die "No LVM volume group defined for activation"
 	fi
-	lvm vgchange -a y $VOLUME_GROUP \
-		|| die "$VOLUME_GROUP: unable to activate volume group"
+	lvm vgchange -a y $VOLUME_GROUP ||
+		die "$VOLUME_GROUP: unable to activate volume group"
+else
+	DEBUG "No LVM volume group defined for activation"
 fi
 
 DEBUG "$(pcrs)"
 
 # LUKS Key slot 0 is the manual recovery pass phrase
 # that they user entered when they installed OS,
 # key slot 1 is the one that we've generated.
-read -s -p "Enter disk recovery key: " disk_password
-echo -n "$disk_password" > "$RECOVERY_KEY"
+read -s -p "Enter Disk Recovery Key/passphrase: " disk_password
+echo -n "$disk_password" >"$RECOVERY_KEY"
 echo
 
-read -s -p "New disk unlock password for booting: " key_password
+read -s -p "New TPM Disk Unlock Key passphrase for booting: " key_password
 echo
-read -s -p "Repeat unlock code: " key_password2
+read -s -p "Repeat TPM Disk Unlock key passphrase for booting: " key_password2
```

Comment:
Nitpick - the new wording is great, but I suggest capitalizing "Key" here to match above (IMO treating "TPM Disk Unlock Key" as a feature name)

Reply:
Fixed in 8809588a :+1: